lib/config.js

```diff
@@ -156,7 +156,8 @@ Config.prototype.defaults = {
     return scheme + '://' + this.get('host') + ':' + this.get('port') + '/';
   },
   parallel: 1,
-  reporter: 'tap'
+  reporter: 'tap',
+  bail_on_uncaught_error: true
```

Comment:
While this is technically a breaking change, I think its a better default and I'm currently preparing a `1.0` so small breaking changes should be fine.